apps/amiapp_flutter/lib/src/screens/inline_messages.dart

```diff
@@ -0,0 +1,123 @@
+import 'package:customer_io/customer_io_widgets.dart';
+import 'package:flutter/material.dart';
+
+import '../components/container.dart';
+
+class InlineMessagesScreen extends StatelessWidget {
+  const InlineMessagesScreen({super.key});
+
+  @override
+  Widget build(BuildContext context) {
+    return AppContainer(
+      appBar: AppBar(
+        title: const Text('Inline Message Test'),
+      ),
+      body: SingleChildScrollView(
+        padding: const EdgeInsets.all(16.0),
+        child: Column(
+          children: [
+            // Top inline message
+            _buildInlineMessageSection('inline-top'),
+            
+            // Skeleton loading content 1
+            _buildSkeletonContent(),
+            
+            // Middle inline message
+            _buildInlineMessageSection('inline-middle'),
+            
+            // Skeleton loading content 2
+            _buildSkeletonContent(),
+            
+            // Bottom inline message
+            _buildInlineMessageSection('inline-bottom'),
+          ],
+        ),
+      ),
+    );
+  }
+
+  Widget _buildInlineMessageSection(String elementId) {
+    return Builder(
+      builder: (context) {
+        return SizedBox(
+          height: 120, // Good height for inline messages
```

Comment:
Would be nice to leave a TODO as a reminder to update this.

Reply:
This is the ticket i am working on now :D